components/widgets/inputs/users_emails_input.tsx

```diff
@@ -25,7 +29,79 @@ import {isGuest} from 'utils/utils';
 
 import './users_emails_input.scss';
 
-export default class UsersEmailsInput extends React.PureComponent {
+import {UserProfile} from 'mattermost-redux/types/users';
+
+const AsyncSelect = require('react-select/lib/Async').default as React.ElementType<AsyncSelectProps<AppSelectOption>>; // eslint-disable-line global-require
```

Comment:
How come we are changing the import here? I don't think this is how we should be importing things.